packages/utils/node-resolver-core/src/NodeResolver.js

```diff
@@ -538,11 +538,12 @@ export default class NodeResolver {
         let moduleDir = path.join(dir, 'node_modules', moduleName);
         let stats = await this.fs.stat(moduleDir);
         if (stats.isDirectory()) {
+          const realModuleDir = this.options.inputFS.realpathSync(moduleDir);
           return {
             moduleName: moduleName,
             subPath: subPath,
-            moduleDir: moduleDir,
-            filePath: path.join(dir, 'node_modules', filename),
+            moduleDir: realModuleDir,
+            filePath: path.join(realModuleDir, subPath || ''),
```

Comment:
```suggestion
            moduleDir: realModuleDir ?? moduleDir,
            filePath: path.join(realModuleDir ?? moduleDir, subPath || ''),
```

Reply:
@mischnic This might solve the issue. The fix can return to its traditional mode when `realModuleDir` is not found for some reason.